appstudio-utils/util-scripts/lib/fetch/cluster.sh

```diff
@@ -55,7 +55,7 @@ save-policy-config() {
     fi
   else
     local non_blocking_data_file
-    non_blocking_data_file="$( json-data-file config policy non_blocking_checks)"
+    non_blocking_data_file=="$DATA_DIR/config.json"
```

Comment:
Oops, will fix this ==

Reply:
Is there even a reason for the variable? Below could just run
```
echo "$non_blocking_data" | jq > "$DATA_DIR/config.json"
```

Reply:
nvm. I see it's used later on in the function also.